tests/mgr/test_cephmgr_validate_insights.py

```diff
@@ -54,6 +58,40 @@ def _ceph_run_insight(node, step_to_validate):
                 raise CephInsightOpsFailure(
                     f"Insights data doesn't have {section} section"
                 )
+    if step_to_validate == "verify-prune-health":
+        # Verify whether the prune-health command deletes the report generated
+        prune_timeout = 0  # Delete all logs
+        out = CephAdm(node).ceph.insights(prune=True, hours=prune_timeout)
+        if out:
+            CephInsightOpsFailure("Failed to prune ceph insights report")
+
+        # Generate report and validate it's not same as earlier
+        out = CephAdm(node).ceph.insights()
+        if out:
+            CephInsightOpsFailure("Failed to get ceph insights report after prune")
+        new_insights_data = json.loads(out)
+        if new_insights_data == insight_data:
+            CephInsightOpsFailure("The insights report has not been cleared")
+
+    if step_to_validate == "verify-log":
+        # Verify the ceph.audit.log has health history recorded and is not consuming more space
+        # and Excessive logging is not found
+        logs, _ = node.exec_command(cmd="cat /var/log/ceph/cephadm.log")
```

Comment:
this log file could be a huge. instead of cat, download to temp location and read.